src/Mizar/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 The Orbit Authors. All rights reserved.
```

Comment:
I think you forgot to add the Mizar folder to the main CMakeLists.txt, as a result the code does not get build by the CI

Reply:
True.